src/hooks/useProAmmPositions.ts

```diff
@@ -197,7 +197,7 @@ export const useFarmPositions = () => {
           .flat()
       })
       .flat()
-      .filter(item => item.liquidity.gt(0))
+    // .filter(item => item.liquidity.gt(0))
```

Comment:
remove or not

Reply:
removed 
this is pre-implementation, we dont have farm contract yet. will have an other version tomorrow

Reply:
btw, I removed it